go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/aquasecurity/trivy-kubernetes
 
-go 1.24
+go 1.24.2
```

Comment:
@afdesk I think we'll have to omit the patch release here as including it causes Trivy to get bumped to the patch release. In trivy we don't pin to a patch release afaik.